src/main/java/cs2103/v15_1j/jimjim/AddTaskCommand.java

```diff
@@ -19,9 +19,9 @@ public String undo(List<TaskEvent> displayList, Storage storage, Searcher search
 
 	@Override
 	public String execute(List<TaskEvent> displayList, Storage storage, Searcher searcher) {
-	    if (storage.create(task)) {
+		displayList.add(task);
```

Comment:
Why do we need two lists?

On Friday, 4 March 2016, Tran Tien Dat <notifications@github.com> wrote:

> In src/main/java/cs2103/v15_1j/jimjim/AddTaskCommand.java
> <https://github.com/cs2103jan2016-v15-1j/main/pull/45#discussion_r54915618>
> :
>
> > @@ -19,9 +19,9 @@ public String undo(List<TaskEvent> displayList, Storage storage, Searcher search
> >
> >  	@Override
> >  	public String execute(List<TaskEvent> displayList, Storage storage, Searcher searcher) {
> > -	    if (storage.create(task)) {
> > +		displayList.add(task);
>
> I think we should have 2 list. One is for display, one is for internal
> working.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cs2103jan2016-v15-1j/main/pull/45/files#r54915618>.
>


-- 
Shaun Thium (Mr.)
*National University of Singapore*
B.Comp (Computer Science) | University Scholar's Programme